python/opentracing/Dockerfile.client

```diff
@@ -0,0 +1,12 @@
+FROM python:3.7
+RUN pip install flask pymongo redis requests sqlalchemy
+RUN pip install lightstep
+RUN pip install opentelemetry-launcher
+RUN pip install git+https://github.com/ocelotl/opentelemetry-python.git@issue_305#subdirectory=opentelemetry-api
+RUN pip install git+https://github.com/ocelotl/opentelemetry-python.git@issue_305#subdirectory=opentelemetry-sdk
+RUN pip install git+https://github.com/ocelotl/opentelemetry-python.git@issue_305#subdirectory=opentelemetry-instrumentation
+RUN pip install git+https://github.com/ocelotl/opentelemetry-python.git@issue_305#subdirectory=instrumentation/opentelemetry-instrumentation-opentracing-shim
+RUN pip install git+https://github.com/ocelotl/opentelemetry-python.git@issue_305#subdirectory=instrumentation/opentelemetry-instrumentation-requests
```

Comment:
Any reason these are being installed from a fork of otel?

Reply:
I still have pending changes that are in open PRs, not yet merged into master. I tried using branches from the main `opentelemetry-python` repo, but they were not present there.